templates/node/src/client.ts.twig

```diff
@@ -1,8 +1,11 @@
 import { fetch, FormData, File } from 'node-fetch-native-with-agent';
 import { createAgent } from 'node-fetch-native-with-agent/agent';
 import { Models } from './models';
+import { Payload } from './payload';
+import * as multipart from 'parse-multipart-data';
+const { buffer } = require('node:stream/consumers');  
```

Comment:
Is this nessessary? Cant we do it without library? DOes it convert from X to buffer? what is X?